test/integration/targets/docker_swarm_service/tasks/tests/options.yml

```diff
@@ -205,6 +205,43 @@
       - constraints_2 is not changed
       - constraints_3 is changed
 
+####################################################################
+## command #########################################################
+####################################################################
+
+- name: command
+  docker_swarm_service:
+    name: "{{ service_name }}"
+    image: alpine:3.8
+    command: '/bin/sh -v -c "sleep 10m"'
+  register: command_1
+
+- name: command (idempotency)
+  docker_swarm_service:
+    name: "{{ service_name }}"
+    image: alpine:3.8
+    command: '/bin/sh -v -c "sleep 10m"'
```

Comment:
Maybe provide the same command as a list here.

Reply:
Yes, always good with more tests.